src/azure-cli/azure/cli/command_modules/backup/custom_wl.py

```diff
@@ -46,8 +46,9 @@
             'sapasedatabase': 'SAPAseDatabase'}
 
 protectable_item_type_map = {'SQLDatabase': 'SQLDataBase',
-                             'HANADataBase': 'SAPHanaDatabase',
+                             'SAPHanaDatabase': 'SAPHanaDatabase',
```

Comment:
This looks like a breaking change. If someone has used HANADataBase as protectable item type in their script, it will start failing. Better to keep the older one and add another entry to the map